### PR TITLE
Fixed building on RHEL-7/CentOS 7

### PR DESCRIPTION
**Pull request by yarda**
Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

### CODE REVIEW

**Comment by jmencak** [comment]
Tested on CentOS7 and UBI7.8, works fine.  Thanks!

**Comment by jmencak** [comment]
Retested, works fine and no longer pulls python3 deps.  Thanks!

File: tuned.spec [hunk 1]
```diff
@@ -60,6 +61,10 @@ Requires(preun): systemd
 Requires(postun): systemd
 BuildRequires: %{_py}, %{_py}-devel
 # BuildRequires for 'make test'
+# python-mock is needed for python-2.7, but it's not available on RHEL-7
+%if %{without python3} && 0%{?rhel} < 8
+BuildRequires: %{_py}-mock
```
Comment (olysonek) [inline tuned.spec:66]: I don't get it. Why are you requiring python-mock on RHEL-7 if it's not available there?

Comment (yarda) [reply tuned.spec:66]: It should be fixed now.

**Comment by yarda** [comment]
Thanks, merging.

**Pull request merged**